.vscode/settings.json

```diff
@@ -14,7 +14,18 @@
         "${workspaceFolder}/src/lib/**",
         "${workspaceFolder}/src/system/**",
         "${workspaceFolder}/third_party/nlassert/repo/include/**",
-        "${workspaceFolder}/third_party/nlio/repo/include/**"
+        "${workspaceFolder}/third_party/nlio/repo/include/**",
+        "${workspaceFolder}/darwin/Framework/CHIP/**",
+        "${workspaceFolder}/src/messaging/**",
+        "${workspaceFolder}/src/protocols/**",
+        "${workspaceFolder}/src/tracing/**",
+        "${workspaceFolder}/src/transport/**",
+        "${workspaceFolder}/src/inet/**",
+        "${workspaceFolder}/src/credentials/**",
+        "${workspaceFolder}/src/data_model/**",
```

Comment:
We don't have an `src/data_model`?

Reply:
https://github.com/project-chip/connectedhomeip/pull/33582